pkg/utils/util.go

```diff
@@ -79,6 +79,11 @@ const (
 	// and used at snapshot content deletion time.
 	AnnDeletionSecretRefName      = "snapshot.storage.kubernetes.io/deletion-secret-name"
 	AnnDeletionSecretRefNamespace = "snapshot.storage.kubernetes.io/deletion-secret-namespace"
+
+	// Annotation for secret name and namespace will be added to the content
+	// and used at snapshot content listing time.
+	AnnListingSecretRefName      = "snapshot.storage.kubernetes.io/listing-secret-name"
+	AnnListingSecretRefNamespace = "snapshot.storage.kubernetes.io/listing-secret-namespace"
```

Comment:
s/listing-secret-name/list-secret-name

On line 56-57, we have:
prefixedSnapshotterSecretNameKey      = csiParameterPrefix + "snapshotter-secret-name"
prefixedSnapshotterSecretNamespaceKey = csiParameterPrefix + "snapshotter-secret-namespace"

Those are used for the deletion secrets.  We need to add something similar for list snapshots, i.e., snapshotter-list-secret-name.

Take a look here: https://github.com/kubernetes-csi/external-provisioner/blob/master/pkg/controller/controller.go#L541
In the external-provisioner, there are multiple secrets being used.  We need to add a parameter in getSecretReference() so that it can handle different secrets.

Reply:
@xing-yang Thank you, and sorry I misunderstood about how to implement...
I created a new PR https://github.com/kubernetes-csi/external-snapshotter/pull/252.
Could you review this PR?